user/pages/03.commerce2/01.getting-started/02.getting-help/docs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Getting help
```

Comment:
I agree about linking to https://drupal.stackexchange.com/. For instance, Ryan is 💯 there whereas Bojan and I are more active on Slack.